lib/icingadb/redisconnection.hpp

```diff
@@ -113,10 +113,14 @@ namespace icinga
 		 */
 		struct WriteQueueItem
 		{
+			QueryPriority Priority;
+			uint_fast64_t PriorityOrder;
```

Comment:
Why "PriorityOrder"? The order has literally nothing to do with the priory member just above.